application/app/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Products\ProductIndexRequest;
+use App\Models\Product;
+use App\Models\ProductCategory;
+
+class ProductController extends Controller
+{
+    /**
+     * @param ProductIndexRequest $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function index(ProductIndexRequest $request)
+    {
+        $product = Product::select([
+            "products.id",
+            "products.name",
+            "products.product_category_id",
+            "products.price",
+            "products.image_path",
+        ]);
```

Comment:
```suggestion
        $product = Product::query();
```
いままでのもそうなんですけど'select'はいらないです
外部に見せたくないカラム(パスワードなど)は`hidden`で隠蔽できるので